deployments/hadoop-yarn/ansible/27-install-zeppelin.yml

```diff
@@ -200,6 +200,18 @@
                     <description>Enable directory listings on server.</description>
                 </property>
 
+                <property>
+                    <name>zeppelin.interpreter.exclude</name>
+                    <value>angular,livy,alluxio,file,psql,flink,ignite,lens,cassandra,geode,kylin,elasticsearch,scalding,jdbc,hbase,bigquery,beam,groovy,flink-cmd,hazelcastjet,influxdb,java,jupyter,kotlin,ksql,mongodb,neo4j,pig,r,sap,spark-submit,sparql,submarine</value>
+                    <description>All the inteprreters that you would like to exclude. You can only specify either 'zeppelin.interpreter.include' or 'zeppelin.interpreter.exclude'. Specifying them together is not allowed.</description>
+                </property>
```

Comment:
Why are these excluded ?
In might be better to explicitly list the interpreters we **do** want in `zeppelin.interpreter.include` rather than excluding an arbitrary list in `zeppelin.interpreter.exclude`.

I've created an issue to follow this up #593.